models/DataObject/ClassDefinition/Data/Numeric.php

```diff
@@ -35,6 +35,20 @@ class Numeric extends Data implements ResourcePersistenceAwareInterface, QueryRe
 
     const DECIMAL_PRECISION_DEFAULT = 0;
 
+    /**
+     * @var array
+     */
+    public static $validFilterOperators = [
```

Comment:
Can this be a `const`?

Reply:
copied from https://github.com/pimcore/pimcore/blob/10.5/models/DataObject/ClassDefinition/Data.php#L104

Reply:
I'd prefer `const` here as well, yes .... 
and probably we should change https://github.com/pimcore/pimcore/blob/10.5/models/DataObject/ClassDefinition/Data.php#L104 as well, as we are working on a major version anyway. 

@sigere can you update your PR accordingly, please 😊